components/KnowledgePanel.css

```diff
@@ -5,3 +5,15 @@
     width: -moz-fit-content;
     padding: 8px;
 }
+
+.header-subtitle{
+    font-size: 20px;
+    margin-top: 8px;
+}
+
+.header-title {
+    font: sans-serif;

```

Comment:
```suggestion
```
Remove this, as it's not a valid value for `font`, which you can tell by doing inspect element and looking at the styles. If we do change the font, we'll change it for the entire knowledge panel rather than just the title.

![chrome_56TThyVCTz](https://user-images.githubusercontent.com/7295783/113033849-55e8f200-9146-11eb-9e01-da3f9f8864c6.png)